node/stateSync.go

```diff
@@ -171,10 +172,14 @@ func (s *stateSync) run() error {
 
 func (s *stateSync) startWorkerThread() {
 	var workerId uint32
-	peersNum := len(s.peers)
-	for workerId = 0; workerId < uint32(peersNum*concurrentSyncRequestPerNeighbor); workerId++ {
+	peersNum := uint32(len(s.peers))
```

Comment:
Are you missing `concurrentSyncRequestPerNeighbor` here?

Reply:
Fixed